Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm

```diff
@@ -155,8 +155,7 @@ static bool experimentalFeatureEnabled(const String& key, bool defaultValue = fa
             resourceLoadStatisticsManualPrevalentResource = WebCore::RegistrableDomain { url };
     }
 #if !RELEASE_LOG_DISABLED
-    static NSString * const WebKitLogCookieInformationDefaultsKey = @"WebKitLogCookieInformation";
-    shouldLogCookieInformation = [defaults boolForKey:WebKitLogCookieInformationDefaultsKey];
+    shouldLogCookieInformation = [defaults boolForKey:@"WebKitLogCookieInformation"];
```

Comment:
Does this notation create a static string, or a new one each time the function is entered?

Reply:
Should be static. 

C compilers make a read only copy and the pointers are passed.

I imagine NSString literals are just Plain Old Data